src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -215,11 +220,11 @@ describe('LSP3UniversalProfile', () => {
   describe('baseContract deployment flag', () => {
     describe('Deployment with all baseContract flags set to false', () => {
       it('Should not deploy base contracts', async () => {
-        await testUPDeploymentWithBaseContractFlag(
+        await testUPDeployment(
```

Comment:
Wouldn't it be good to add an extra assertion check here in this test, to make sure the function behave as expected?


(Just an idea) For instance, to make sure that no new base contracts were deployed,
you could test that the bytecode of the proxy includes the address of the contract address specified in `versions.json`?

Reply:
`testUPDeployment` already checks that the correct base contracts have been deployed (or not deployed if they shouldn't be). Adding the proxy address check would be a nice addition though